tests/unittests/graphOptzTest.cpp

```diff
@@ -2423,3 +2423,28 @@ TEST_F(GraphOptz, sinkTransposeBelowChannelShuffleNodesAndEliminate) {
   EXPECT_EQ(preShuffleRN->getDims(),
             llvm::makeArrayRef(expectedPreShuffleRNDims));
 }
+
+/// Test that convertPlaceholdersToConstants works properly with quantized
+/// types.
+TEST_F(GraphOptz, QuantizedFC) {
+  auto *input = mod_.createPlaceholder(ElemKind::Int8QTy, {2, 32}, 1.0, 0,
+                                       "input", false);
+  auto *weights = mod_.createPlaceholder(ElemKind::Int8QTy, {32, 32}, 1.0, 0,
+                                         "weights", false);
+  auto *bias =
+      mod_.createPlaceholder(ElemKind::Int32QTy, {32}, 1.0, 0, "bias", false);
+  auto *output = mod_.createPlaceholder(ElemKind::Int8QTy, {2, 32}, 1.0, 0,
+                                        "output", false);
+
+  auto *fc = F_->createFullyConnected("fc", input, weights, bias);
+  F_->createSave("save", fc, output);
+
+  ctx_.allocate(input);
+  ctx_.allocate(weights);
+  ctx_.allocate(bias);
+  ctx_.allocate(output);
+
+  glow::convertPlaceholdersToConstants(F_, ctx_, {input, output});
+  EXPECT_EQ(mod_.getConstants().size(), 2);
```

Comment:
can you comment on what are constants (weight/bias)
and on placeholders (i assume it's input/output, what else?)

Reply:
Yup, weights/bias are constants.  All 4 things are still placeholders in the module.  Even though weight/bias have been converted, they haven't been removed from the module.  I don't know how to make them go away :-).  I'll add comments to this effect.